<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/bank-icon</normaloff>:/icons/bank-icon</iconset>
  </property>
  <property name="locale">
   <locale language="German" country="Germany"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>681</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hier können die Einstellungen von AB-Transfers verändert werden.&lt;br /&gt;Wenn eine Einstellung noch nicht unterstützt wird ist sie deaktiviert, zeigt aber das aktuelle Verhalten richtig an.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>421</width>
     <height>131</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>121</height>
    </size>
   </property>
   <property name="title">
    <string>Speicherorte / Dateien</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,5,0">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Standart-Ordner</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>lineEdit_dataDir</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_dataDir">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>22</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Legt das Standart Verzeichniss zum Speichern von Daten fest.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="toolButton_selectDataDir">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Aktuelle Daten</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>lineEdit_accountData</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_accountData">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>22</height>
       </size>
      </property>
      <property name="toolTip">
       <string>In dieser Datei werden die aktuellen Daten gespeichert.

Dies umfasst die Konto-Salden von allen Accounts sowie
die Daueraufträge und terminierten Überweisungen.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="toolButton_selectAccountData">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Historie</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>lineEdit_history</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_history">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>22</height>
       </size>
      </property>
      <property name="toolTip">
       <string>In dieser Datei wird die Historie gespeichert.

Alle Aufträge die zum Institut übertragen wurden
werden in dieser Datei gespeichert.</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="toolButton_selectHistory">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Empfänger</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>lineEdit_recipients</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_recipients">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>22</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Alle bekannten Empfänger werden in dieser Datei gespeichert.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QToolButton" name="toolButton_selectRecipients">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_debugInfoDialog">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Wenn deaktiviert wird das Debug-Info Fenster deaktiviert und
die Debug-Ausgaben finden nur noch auf der Konsole statt.

Wenn aktiviert können alle Meldungen die normalerweise nur
auf der Konsole stattfinden auch in dem Debug-Info Fenster
angezeigt werden.</string>
   </property>
   <property name="text">
    <string>Debug Meldungen im Debug-Dialog anzeigen</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Hinweise anzeigen</string>
   </property>
   <widget class="QCheckBox" name="checkBox_warnCosts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Wenn aktiviert wird ein Hinweis zu eventuellen Kosten
beim starten des Programms angezeigt.</string>
    </property>
    <property name="text">
     <string>eventuelle Kosten</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_JobAddedToOutput">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Wenn aktiviert wird jedes mal wenn ein Auftrag erfolgreich
zum Ausgang hinzugefügt wurde ein entsprechender
Hinweis angezeigt.</string>
    </property>
    <property name="text">
     <string>Auftrag zum Ausgang hinzugefügt</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Wenn eine Transaktion durchgeführt wurde und der
Empfänger noch nicht in den bekannten Empfängern
vorhanden ist wird dieser automatisch zu diesen
hinzugefügt.</string>
   </property>
   <property name="text">
    <string>Automatisch die bekannten Empänger erweitern</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Aktualisierung beim Starten</string>
   </property>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Wenn aktiviert wird automatisch beim starten des
Programms für alle Konten eine Aktualisierung des
Kontostandes in den Ausgang eingestellt.</string>
    </property>
    <property name="text">
     <string>Kontostände</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Wenn aktiviert wird automatisch beim starten des
Programms für alle Konten eine Aktualisierung der
Daueraufträge in den Ausgang eingestellt.</string>
    </property>
    <property name="text">
     <string>Daueraufträge</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Wenn aktiviert wird automatisch beim starten des
Programms für alle Konten eine Aktualisierung der
terminierten Überweisungen in den Ausgang eingestellt.</string>
    </property>
    <property name="text">
     <string>Terminierte Überweisungen</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_dataDir</tabstop>
  <tabstop>toolButton_selectDataDir</tabstop>
  <tabstop>lineEdit_accountData</tabstop>
  <tabstop>toolButton_selectAccountData</tabstop>
  <tabstop>lineEdit_history</tabstop>
  <tabstop>toolButton_selectHistory</tabstop>
  <tabstop>lineEdit_recipients</tabstop>
  <tabstop>toolButton_selectRecipients</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
